climesync/commands.py

```diff
@@ -1207,6 +1262,15 @@ def get_users(post_data=None, role=None):
         users = [user for user in users
                  if user["meta"] and meta in user["meta"].upper()]
 
+    if interactive:
```

Comment:
This block pops up a few times, might want to consider factoring it out somehow.  Not a requirement.